accelerator/apis.c

```diff
@@ -96,6 +96,17 @@ status_t api_find_transactions_by_tag(
     goto done;
   }
 
+  char req_tag[NUM_TRYTES_TAG + 1];
+
+  if (strlen(obj) == NUM_TRYTES_TAG) {
+    req_tag = obj;
+  } else if (strlen(obj) < NUM_TRYTES_TAG) {
```

Comment:
Don't call `strlen` on the same object since it would not be changed within these statements.